.github/workflows/srt.yml

```diff
@@ -40,32 +39,13 @@ jobs:
       - id: load-env
         run: |
           sudo apt-get update
-          sudo apt-get install libxml2-utils pylint wget gfortran openmpi-bin netcdf-bin libopenmpi-dev cmake libnetcdf-dev
+          sudo apt-get install libxml2-utils wget gfortran openmpi-bin netcdf-bin libopenmpi-dev cmake libnetcdf-dev pylint
```

Comment:
Based on the title of the PR, shouldn't pnetcdf be in apt-get call?  But looking at you commit comments, you're not using pnetcdf.  So the title of the PR should be updated.